### DISCUSSION TITLE
Broken character boxes appears when using dp.Table(df)

### DISCUSSION THREAD

**Discussion by aleiyoo** [datapane/datapane#63] [Help]
When using dp.Table(df), I encountered the Chinese character scrambling problem. How to solve it?
![image](https://user-images.githubusercontent.com/76715957/103726151-2927b180-5013-11eb-916b-45b6351778f6.png)

**Comment by mands** [comment]
Hi, I think this is may be locale issue - can you confirm what browser, OS, and language you are using? Also do you happen to know if the text in your report Unicode?

**Comment by JoeBuzh** [comment]
I got the same problem, but when I published in website using DataTable, it seems to work correctly.![image](https://user-images.githubusercontent.com/23374841/115215673-51f13400-a136-11eb-8487-26ebca9e9f44.png)![image](https://user-images.githubusercontent.com/23374841/115215790-6b927b80-a136-11eb-8efe-d764c35efa40.png)
The report website url is https://datapane.com/u/joebu12/reports/offshore-lpf-dismiss/#

**Reply by JoeBuzh** [reply]
local html file tested in Chrome, Firefox, Safari, code by python3.6
`
dp.Report(blocks=blocks0).save(
    path=r'OffShore_LPF_Dismiss.html', 
    open=True)
`
